Classes/Outlet/Pipe/EmailPipe.php

```diff
@@ -45,19 +47,25 @@ class EmailPipe extends AbstractPipe implements PipeInterface
     protected $body = null;
 
     /**
+     * @var string|NULL
```

Comment:
Should have a brief description since it isn't as obvious as the other properties.